README.md

```diff
@@ -77,53 +96,85 @@ perl table_annovar.pl input/test_VEP.vcf hg38 --buildver hg38 --out test_VEP --r
 python InterVar.py -b hg38 -i input/test_VEP.vcf --input_type=VCF -o test_VEP
 ```
 4. Run [AutoPVS1 v2.0](https://github.com/JiguangPeng/autopvs1/releases/tag/v2.0).
-5.  Optional: provide a ClinVar VCF file. If not supplied by the user, the most recent ClinVar file will be downloaded with `download_db_files.sh` and used in AutoGVP.
-6. Run AutoGVP.
+5. Optional: provide a ClinVar VCF file. If not supplied by the user, the most recent ClinVar file will be downloaded with `download_db_files.sh` and used in AutoGVP.
+6. Download required db files
+7. Run `select-clinVar-submissions.R` within `AutoGVP/`
```

Comment:
should we also note that if this has been run before and same ClinVar data is used, this doesn't need to run every time?